### PR TITLE
Script running

### PR DESCRIPTION
**Pull request by jorisoudejans**
Created a base for a running script in the back-end.

### CODE REVIEW

File: app/assets/app/api/script.service.ts [hunk 1]
```diff
@@ -9,18 +9,23 @@ export class ScriptService {
     private _heroesUrl = "api/scripts";  // URL to web api
     getScripts (): Observable<Script[]> {
         return this.http.get(this._heroesUrl)
-            .map(this.extractData)
-            .catch(this.handleError);
+            .map(ScriptService.extractData)
+            .catch(ScriptService.handleError);
```
Comment (hiddely) [inline app/assets/app/api/script.service.ts:13]: Why did you make this static?

File: conf/routes [hunk 2]
```diff
@@ -3,24 +3,25 @@
...
+GET     /api/scripts/:id                @controllers.Script.get(id: Long)
+POST    /api/scripts/create             @controllers.Script.create()
+POST    /api/scripts/:id/actions/create     @controllers.Script.addAction(id: Long)
 
 # Live script
-GET     /api/scripts/:id/status             controllers.Script.status(id: Long)
-GET     /api/scripts/:id/update             controllers.Script.updateActiveScript(id: Long)
+GET     /api/scripts/status                 @controllers.Script.getActiveScript()
+GET     /api/scripts/:id/start              @controllers.Script.startScript(id: Long)
+GET     /api/scripts/:id/update             @controllers.Script.updateActiveScript(id: Long)
```
Comment (skoppers) [inline conf/routes:19]: why did you add @ everywhere?

Comment (hiddely) [inline conf/routes:19]: Its to fix the IDE errors for the method calls

**Comment by feclist** [comment]
Why is ngOnInit() { this.startScript(1); } hardcoded in app.component.ts? This can be changed to a getter for the active script.

**Comment by hiddely** [comment]
Essential features! Now to merge...

File: app/assets/app/api/script.service.ts (hunk 1 above)
Comment (jorisoudejans) [inline app/assets/app/api/script.service.ts:13]: Cuz Checkstyle (or PMD / IntelliJ) told me to do so

**Comment by skoppers** [comment]
oke, looks good :+1:

**Comment by jorisoudejans** [comment]
@feclist StartScript creates an active script. Otherwise getActiveScript returns a 404. When we have the 'select script'-view, this can be unhardcoded.

**Comment by feclist** [comment]
@jorisoudejans That was what I thought indeed
